src/main/resources/db/migration/common/V1_232__Update_TransactionUrgency_Table.sql

```diff
@@ -0,0 +1,10 @@
+ALTER TABLE transcription_urgency ADD COLUMN IF NOT EXISTS display_state BOOLEAN NOT NULL DEFAULT TRUE;
+
+INSERT INTO darts.transcription_urgency (tru_id, description, display_state) VALUES(4, 'Up to 3 working days', true);
+INSERT INTO darts.transcription_urgency (tru_id, description, display_state) VALUES(5, 'Up to 7 working days', true);
+INSERT INTO darts.transcription_urgency (tru_id, description, display_state) VALUES(6, 'Up to 12 working days', true);
```

Comment:
FYI: If you have a default state and that is set to the state you want, you don't need to provide it when you do an insert. So you could have done something like this instead:

`INSERT INTO darts.transcription_urgency (tru_id, description) VALUES(6, 'Up to 12 working days');`

Reply:
Updated as suggested

Reply:
Referring to commit aea23bd7f6a71ae0951bd194b65cc3b708ea2c1c - it would be better if we didn't introduce a `DEFAULT TRUE` since it isn't in our v54 reference schema?